docs/cloud/rahti2/usage/getting_started.md

```diff
@@ -0,0 +1,19 @@
+# Using the web interface
+
+All applications launched in OpenShift run within **projects** that can be
+created by any authenticated user. Each project has its own private virtual
+network and is isolated from other projects. Users can only see projects
+they have created themselves or that have been shared with them. Any
+containers, volumes and other resources created by users are always created
+within a project.
+
+You can either select applications to run in the application catalog that is
+visible when first logging in or launch any applications using the
+primitives described in the [background](../concepts.md) chapter.
+
+Log in at <https://landing.2.rahti.csc.fi/> (see [Getting access](../access.md)
```

Comment:
The landing page will be available in both https://rahti.csc.fi and https://landing.2.rahti.csc.fi/ . which one is better to have in the docs?

Reply:
fixed